src/main/handlers.js

```diff
@@ -7,26 +7,24 @@ const config = require('../config')
 const path = require('path')
 
 function install () {
-  if (process.platform === 'darwin') {
-    installDarwin()
-  }
-  if (process.platform === 'win32') {
-    installWin32()
-  }
-  if (process.platform === 'linux') {
-    installLinux()
+  switch (process.platform) {
+    case 'darwin': installDarwin()
+      break
+    case 'win32': installWin32()
+      break
+    case 'linux': installLinux()
+      break
```

Comment:
Not a fan of `switch` statements. I'll take a small set of `if`s any day. Switches are prone to misbehaving if you forget to `break`.

Reply:
Not the most elegant syntax inherited from C, but it is intended for this scenario. From a logical point of view the elses are missing. Adding them with all the curly brackets is not a very pretty view neither. 

Lets get these pull requests out of the way and focus on important stuff.